python/tests/test_table.py

```diff
@@ -182,3 +182,7 @@ def test_display_when_largest_coord_non_dimensional():
                                        values=np.random.random(10),
                                        unit=sc.units.counts))
     sc.table(da)
+
+
+def test_with_scalar():
+    sc.table(sc.scalar(value=1.0))
```

Comment:
what happens if there are multiple scala variables, e.g. in a Dataset?

Reply:
Good question. I didn't try that. I did with DataArray, but i'll add tests and check outputs for all.